Explorer/Assets/DCL/Backpack/Gifting/Models/EmoteGiftable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DCL.AvatarRendering.Emotes;
+using DCL.AvatarRendering.Loading.Components;
+
+namespace DCL.Backpack.Gifting.Models
+{
+    public readonly struct EmoteGiftable : IGiftable
```

Comment:
Please review this `Emote/Wearable Giftable` as it looks like the specific type is always used as `IAvatarAttachment` and there is no need to split into concrete types